### ISSUE TITLE
Helmet + Armor Disappearing

### ISSUE THREAD

**Issue by BigheadSMZ** [diasurgical/devilutionX#6283]
### Operating System

Windows x64

### DevilutionX version

1.5.0 (latest release)

### Describe

So I have no idea what is causing this, or how its being triggered, but it seems sometimes when I load up my character either through a new game or loading a save, my helmet and armor are gone. The first time it happened, I beat Diablo on normal difficulty, saved just before the fight with him. Started a new game on nightmare, and my helmet and armor were gone. I though maybe I dropped it somewhere, loaded up my save before Diablo, helmet and armor gone. I thought maybe I dropped them or something? Both were uniques obtained from the quests, so I thought maybe there was just some bug with them.

So I just bought a random armor off of Wart, found a new helmet, just basic blue items. Played through nightmare up until level 8, got tired, saved, quit for the night. Loaded up my save file today at level 8, and my helmet and armor are gone again! This time I was absolutely sure I did not drop them anywhere at any point. Is this a known issue or am I just unlucky? Any ideas of what could cause this?

### To Reproduce

I have no idea how to reproduce it. I basically just play for awhile, quit, then load the game up at a later time. And when I check my character, his armor and helmet are not there.

### Expected Behavior

My helmet and armor to not disappear.

**Comment by AJenbo** [comment]
Just to check, your also sure they didn't get damaged to the point of breaking?

**Comment by DakkJaniels** [comment]
Yeah, considering you are doing Nightmare with a low level character, your equipment is probably breaking.

**Comment by BigheadSMZ** [comment]
Yeah I'm and idiot and didn't consider it when I made the issue that my gear may have broke. I could have sworn I saved with them equipped before fighting Diablo although I could be mistaken. I do try to repair at Griswald every trip to town, but I get goofy when I'm tired and I've been mostly playing before I go to bed. It's been years since I played Diablo and I entirely forgot that they disappear when fully breaking. I've been playing around with this game in between Diablo IV sessions, and having a blast with both. Sorry about the possible false report, it's most likely the case that I just wasn't paying attention to durability. I'll close the issue and keep a close eye on it from now just to make sure. If it ends up actually being a bug I'll reopen it. Thanks for your time.

And to the devs since I'm here: thanks so much for this updated engine! I got the GOG version of the game since I only have the original disc and not the DLC and wanted to play it for the first time. It was... functional. But with DevilutionX, it feels like I'm back in the late 90s again which is an amazing feeling.
